daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md

```diff
@@ -62,59 +77,82 @@ The following example shows an XML content in CloudEvent v1.0 serialized as JSON
 }
 ```
 
-### Message subscription
+#### Content types
+
+When publishing a message, it's important to specify the content type of the data being sent.
+Unless specified, Dapr will assume `text/plain`.
 
-Dapr applications can subscribe to published topics. Dapr allows two methods by which your applications can subscribe to topics:
+For Dapr's HTTP API, the content type can be set in a `Content-Type` header.
 
- - **Declarative**, where a subscription is defined in an external file,
- - **Programmatic**, where a subscription is defined in the user code.
+gRPC clients and SDKs have a dedicated content type parameter.
+
+### Message subscription
 
- Both declarative and programmatic approaches support the same features. The declarative approach removes the Dapr dependency from your code and allows for existing applications to subscribe to topics, without having to change code. The programmatic approach implements the subscription in your code.
+Dapr applications can subscribe to published topics via two methods that support the same features: declarative and programmatic.
 
-  For more information read [How-To: Publish a message and subscribe to a topic]({{< ref howto-publish-subscribe >}}).
+| Subscription method | Description |
+| ------------------- | ----------- |
+| **Declarative** | Subscription is defined in an **external file**. The declarative approach removes the Dapr dependency from your code and allows for existing applications to subscribe to topics, without having to change code. |
+| **Programmatic** | Subscription is defined in the **user code**. The programmatic approach implements the subscription in your code. |
 
+For more information, read [about the subscriptions in the how-to]({{< ref howto-publish-subscribe.md >}}).
 
 ### Message delivery
 
-In principle, Dapr considers message successfully delivered when the subscriber responds with a non-error response after processing the message. For more granular control, Dapr's publish/subscribe API also provides explicit statuses, defined in the response payload, which the subscriber can use to indicate the specific handling instructions to Dapr (e.g. `RETRY` or `DROP`). For more information on message routing read [Dapr publish/subscribe API documentation]({{< ref "pubsub_api.md#provide-routes-for-dapr-to-deliver-topic-events" >}})
+In principle, Dapr considers a message successfully delivered once the subscriber processes the message and responds with a non-error response. For more granular control, Dapr's Pub/sub API also provides explicit statuses, defined in the response payload, with which the subscriber indicates specific handling instructions to Dapr (for example, `RETRY` or `DROP`).
```

Comment:
This message delivery section should be moved to the Sending message with Cloud Event section.